orion/sites/admin/src/announce/announce.styl

```diff
@@ -1,13 +1,48 @@
 @import "../static.styl"
 
+
 #view-announce
   width: 100%
   height: 400px
   padding: 0 spacing2x
   box-sizing: border-box
-  color: colorBlackDark
+  color: colorGrayDark
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  .announce-lists
+    display: inline-block
+    list-style: none
+    padding-left: 0
+    height: lineHeight6
+    line-height: lineHeight6
+    border-bottom: 1px solid colorGrayLight
+
+    & .li-small
+      width: spacing8x
+      display: inline-block
```

Comment:
there's a `display: inline-block` in here, .li-med, and .li-large.
Can we use a common className to give all these `.li-x` elements `display: inline-block` so it only appears once?

Reply:
Originally I just had the display:inline-block in the .announce-lists class, which is the "parent" class for .li-med, .li-large, and .li-small. However, this causes formatting issues in that the display: inline-block doesn't appear in the list elements.